### ISSUE TITLE
Breakglass flow in web app

### ISSUE THREAD

**Issue by yolocs** [abcxyz/jvs#296]
### TL;DR

As identified in https://github.com/abcxyz/jvs/pull/294/files#r1253404326

The UI allows `breakglass` category but will fail to generate a breakglass token. One immediate fix is to allow the [processor](https://github.com/abcxyz/jvs/blob/a02ba7d38f4b16c203d37e0d879b6c67883ec43a/pkg/justification/processor.go#L148) to create breakglass token. However, the underlying logic doesn't make sense. Breakglass should be used when JVS is down so the user can bypass it. If JVS web UI is down, there is no easy way for a user to inject a breakglass token (even though they can generate one with JVS CLI) into their web request.

Instead, I'd like to disable breakglass category in JVS web UI for now and figure out how to do breakglass in the web flow.

### Environment Details

```markdown
n/a
```

**Comment by yolocs** [comment]
https://modheader.com/ might be useful
